fdbrpc/LoadPlugin.h

```diff
@@ -20,13 +20,14 @@
 
 #pragma once
 
+// Will need to be defined at link time
+extern "C" void *get_plugin(const char *plugin_type_name_and_version);
```

Comment:
I'd think we should probably rename the TLS plugin's get_plugin to something TLS-specific